packages/main/src/plugin/color-registry.ts

```diff
@@ -840,6 +857,10 @@ export class ColorRegistry {
       dark: colorPalette.white + '2',
       light: colorPalette.black + '2',
     });
+    this.registerColor(`${link}-light`, {
+      dark: colorPalette.gray[200],
+      light: colorPalette.charcoal[900],
+    });
```

Comment:
not sure to understand the meaning of link-light ?

Reply:
This looks like the "More details" button. IMHO it would be better to make the entire card clickable (so this link isn't special/doesn't need its own color) or use Link here vs defining another custom color.

Reply:
OK, I changed to use a `Button type="link"`